src/components/Header/Header.tsx

```diff
@@ -1,54 +1,61 @@
 import React, { useState } from "react";
 // import { Header, Image } from 'semantic-ui-react'
-import Curiologo from "../assets/images/Csmall.png";
-import "./header.css";
+import Curiologo from '../assets/images/Csmall.png'
+import './header.css';
+import { Link ,BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 const HeaderCurio = () => {
   //Used to toggle the hamburger menu
   const [triggerMenu, setTriggerMenu] = useState(false);
 
   return (
     <>
-      <div className="curio__header">
-        <div className="gradient__text">
+
+      <div className='curio__header'>
+          {/* Wrapping the Link components within the Router component */}
+      <Router>
+         {/* Logo linking to the homepage */}
+        <Link to='/'>
+        <div className='gradient__text'>
           <h1>
-            <img src={Curiologo} alt="Curio" className="curio__icon" />
+            <img src={Curiologo} alt='Curio' className='curio__icon' />
             <span>URIO</span>
           </h1>
           <p className="curio__tag">
             <b>Your Video Translator :)</b>
           </p>
         </div>
+        </Link>
 
         <div className="curio__sign">
           <p>About</p>
           <p>Sign In</p>
           <button type="button">Sign Up</button>
         </div>
+        </Router>
       </div>
       {/*Added a mobile version
      Screen width is < 768px above is disabled and below is triggered
      Added hamburger menu and generic options for the menu
      */}
-      {}
-      <div className="curio__header-m">
-        <div className=" gradient__text  gradient__text-m">
+      { }
+      <div className='curio__header-m'>
+         {/* Wrapping the Link components within the Router component */}
+         <Router>
+          {/* Logo linking to the homepage */}
+          <Link to='/'>
```

Comment:
```suggestion
          <Link to="/" style={{ textDecoration: "none" }}>
```

Currently when you hover over the logo, it shows underline. We can use this to avoid that.

Reply:
Inline css added in commit [b1e65d5aae](https://github.com/Curio-org/Curio/pull/67/commits/b1e65d5aae9e07aceaa5602029c8e72898d2fc92)